packages/shorebird_cli/test/src/commands/preview_command_test.dart

```diff
@@ -723,20 +723,18 @@ channel: ${track.channel}
         when(() => argResults.wasParsed('app-id')).thenReturn(false);
         when(() => argResults['app-id']).thenReturn(null);
         when(
-          () => logger.chooseOne<AppMetadata>(
+          () => logger.chooseOne<String>(
             any(),
             choices: any(named: 'choices'),
-            display: any(named: 'display'),
           ),
-        ).thenReturn(app);
+        ).thenReturn('dev');
 
         await runWithOverrides(command.run);
 
         verify(
-          () => logger.chooseOne<AppMetadata>(
-            'Which app would you like to preview?',
+          () => logger.chooseOne<String>(
+            any(),
             choices: any(named: 'choices'),
```

Comment:
We should probably verify that the flavors were provided as arguments to `chooseOne`

Reply:
Fair enought! I've changed it

Reply:
Did you push? I'm still seeing:

```dart
 verify(
          () => logger.chooseOne<String>(
            any(),
```

Reply:
<img width="548" alt="Screenshot 2024-04-03 at 13 52 16" src="https://github.com/shorebirdtech/shorebird/assets/835641/45f14594-df44-4553-a90b-446605519f32">

I pushed yes! That is weird, I am seeing it correctly as shown in the print above

Reply:
Ah! The issue is just that the test only has a single flavor. We should probably update that (and maybe change the command to not prompt if only one flavor is present, although that seems like sort of a silly setup)

Reply:
Oh, yeah, fair point!

Should we do that handling in a different PR though? I feel that this PR is addressing the detecting a shorebird project that has flavors, no matter the number, then auto selecting the only flavor would probably be a change of its own?

Let me know if you agree, otherwise I am happy to update this PR to include this as well.

Reply:
I think the "single flavor" fix can be done in a future PR (if ever), but the test should probably be updated to include more than one flavor